deploy/kubernetes/helm/che/values.yaml

```diff
@@ -62,6 +62,7 @@ global:
   securityContext:
     runAsUser: 1724
     fsGroup: 1724
+  postgresDebug: false
```

Comment:
Maybe it would be better to rename this property to make it clear that it enabled only debug logs of Postgres?
Now it may make an impression that Postgres is started in Debug mode, with additional ports exported...
It's not even suggestion but just thought. I'm OK with current naming as well.

Reply:
@sleshchenko Thanks! Good idea, I've updated my branch to use `postgresDebugLogs` instead.